node/src/validator/mod.rs

```diff
@@ -185,9 +187,9 @@ impl<N: Network, C: ConsensusStorage<N>> Validator<N, C> {
     }
 
     /// Return the BFT consensus handle.
-    #[cfg(feature = "test")]
     pub fn bft(&self) -> &RunningConsensusInstance<BftExecutionState<N, C>> {
-        self.bft.as_ref().unwrap()
+        // Safe: it is used only once it's populated.
+        self.bft.get().unwrap()
```

Comment:
Maybe an `expect` would be nice here?

Reply:
for future-proofing? it's currently impossible to trigger it, but sure, I can do that

Reply:
yeah, mostly—just in case someone tries to call this before it's populated in a few years